src/components/Picker/index.js

```diff
@@ -171,7 +165,9 @@ class Picker extends PureComponent {
                     )}
                     <RNPickerSelect
                         onValueChange={this.onInputChange}
-                        items={this.items}
+
+                        // We add a text color to prevent white text on white background dropdown items on Windows
+                        items={_.map(this.props.items, item => ({...item, color: themeColors.pickerOptionsTextColor}))}
```

Comment:
NAB, but guessing there is no way to pass the `color` to the picker component instead of with each option? Seems a bit weird to have to decorate the items with the color property if they are all going to be the same color.

Reply:
I agree, but unfortunately I don't think there's a way around it or I couldn't find another solution. Here's the link to the [lib docs](https://github.com/lawnstarter/react-native-picker-select#props).

Reply:
No worries that's what I figured. I can't think of any reason why we might want to avoid this other than performance (though definitely not for the case this PR is fixing). But if performance is a concern we can find a more optimal way to recalculate things. Not concerned really.